doc/man/8/ceph.rst

```diff
@@ -361,19 +361,87 @@ fs
 
 Manage cephfs file systems. It uses some additional subcommands.
 
+Subcommand ``add_data_pool`` adds an new data pool to the FS. Ths pool can
+be used for file layouts as an alternate location to store the file data.
+
+Usage::
+
+    ceph fs add_data_pool <fs-name> <pool name/id>
+
+Subcommand ``authorize`` creates a new client that will be authorized for the
+given path in ``<fs_name>``. Pass ``/`` to authorize for the entire FS.
+``<perms>`` below can be ``r``, ``rw`` or ``rwp``.
+
+Usage::
+
+    ceph fs authorize <fs_name> client.<client_id> <path> <perms> [<path> <perms>...]
+
+Subcommand ``dump`` displays the FSMap at the given epoch (default: current).
+This includes all file system settings, MDS daemons and the ranks they hold
+and list of standby MDS daemons.
+
+Usage::
+
+    fs dump [epoch]
```

Comment:
it should be `ceph fs dump [epoch]` right, to make it uniform with other illustrations